usr.sbin/service/service.8

```diff
@@ -129,13 +136,40 @@ option is used, the corresponding variable is set accordingly.
 .Sh EXIT STATUS
 .Ex -std
 .Sh EXAMPLES
-The following are examples of typical usage of the
-.Nm
-command:
+Enable a service, then start it:
+.Bd -literal -offset -indent
+service sshd enable
+service sshd start
+.Ed
+.Pp
+Disable a service, then stop it:
```

Comment:
Unless you wanna show off `onestop` below, you should reverse the phrasing here.

Reply:
are you sure?  usually `stop` would come before `disable`, because otherwise stop won't work.

or do you mean we should show an example of `disable` followed by `onestop`?

Reply:
never mind, i see what you mean.  now fixed.